src/.nuget/BuildRyuJitPackage.ps1

```diff
@@ -0,0 +1,154 @@
+        {
+            continue
+        }
+
+        $date = [datetime]($blob.properties."last-modified")
+        if (!$chosenBlob -or ($chosenDate -and $date -ge $chosenDate))
+        {
+            $chosenBlob = $blob.name
+            $chosenDate = $date
+        }
+    }
+
+    return $chosenBlob
+}
+
+# Get the list of blobs in storage
+$json = (azure storage blob list -a $storageAccount -k $storageKey $storageContainer --json) -join ""
+$blobs = ConvertFrom-Json $json
+
+# Find, fetch, and extract the latest Ubuntu and OSX blobs
+[System.Reflection.Assembly]::LoadWithPartialName("System.IO.Compression.FileSystem") | Out-Null
+
+$ubuntuBlob = Get-Latest-Blob-Name $blobs "Ubuntu.14.04_CoreClr_x64_Release_Enu.zip"
+$osxBlob = Get-Latest-Blob-Name $blobs "OSX_CoreClr_x64_Release_Enu.zip"
+
+if (!$ubuntuBlob)
+{
+    Write-Error "Could not locate an Ubuntu drop in Azure."
+    exit 1
+}
+
+if (!$osxBlob)
+{
+    Write-Error "Could not locate an OS X drop in Azure."
+    exit 1
+}
+
+azure storage blob download -m -q -a $storageAccount -k $storageKey $storageContainer $ubuntuBlob
+if ($LastExitCode -ne 0)
+{
+    Write-Error "Failed to fetch Ubuntu drop $ubuntuDrop from Azure."
+    exit 1
+}
+
+azure storage blob download -m -q -a $storageAccount -k $storageKey $storageContainer $osxBlob
+if ($LastExitCode -ne 0)
+{
+    Write-Error "Failed to fetch OS X drop $osxBlob from Azure."
+    exit 1
+}
+
+$ubuntuDirectory = [System.IO.Path]::GetFileNameWithoutExtension($ubuntuBlob)
+try
+{
+    [System.IO.Compression.ZipFile]::ExtractToDirectory($ubuntuBlob, $ubuntuDirectory)
+}
+catch
+{
+    Write-Error "Failed to extract Ubuntu drop to $ubuntuDirectory`: $_.Exception.Message"
+    exit 1
+}
+
+$osxDirectory = [System.IO.Path]::GetFileNameWithoutExtension($osxBlob)
+try
+{
+    [System.IO.Compression.ZipFile]::ExtractToDirectory($osxBlob, $osxDirectory)
+}
+catch
+{
+    Write-Error "Failed to extract OS X drop to $osxDirectory`: $_.Exception.Message"
+    exit 1
+}
+
+# Gather the bits from the Ubuntu and OSX blobs into the bin directory
+Copy-Item -Path @("$ubuntuDirectory\libryujit.so", "$osxDirectory\libryujit.dylib") -Destination $binariesDir
+if ($LastExitCode -ne 0)
+{
+    Write-Error "Failed to copy cross-platform bits to $binariesDir."
+    exit 1
+}
+
+# Gather the .nuspecs and their dependencies into the package output directory
+$files = @(
+    "$nuspecDir\Microsoft.DotNet.RyuJit.nuspec",
+    "$nuspecDir\runtime.json",
+    "$nuspecDir\toolchain.osx.10.10-x64.Microsoft.DotNet.RyuJit.nuspec",
+    "$nuspecDir\toolchain.ubuntu.14.04-x64.Microsoft.DotNet.RyuJit.nuspec"
+)
+Copy-Item -Path $files -Destination $packageOutputDir
+if ($LastExitCode -ne 0)
+{
+    Write-Error "Failed to copy nuspecs to $packageOutputDir."
+    exit 1
+}
+
+# Create the packages.
+$packages = @(
+    "toolchain.osx.10.10-x64.Microsoft.DotNet.RyuJit",
+    "toolchain.ubuntu.14.04-x64.Microsoft.DotNet.RyuJit",
+    "Microsoft.DotNet.RyuJit"
```

Comment:
Not publishing "toolchain.win7-x64.Microsoft.DotNet.RyuJit" here is intentional here? So, I assume window one is published as part of msbuild (without this redirection package)?